src/main/webapp/components/basic-search/basic-search.js

```diff
@@ -233,26 +233,42 @@ const defaultFilters = {
   [LOCATION_KEY]: makeDefaultSearchGeo(),
 }
 
-const getFilterMap = props => {
-  if (props.query && props.query.filterTree) {
-    return fromFilterTree(props.query.filterTree, {
-      basicSearchMatchType: props.basicSearchMatchType,
+const getFilterMap = ({ query, basicSearchMatchType }) => {
+  if (!query) {
+    return Map({ text: '*', sources: undefined })
+  }
+
+  let filterMap = query.filterTree
+    ? fromFilterTree(query.filterTree, {
+        basicSearchMatchType: basicSearchMatchType,
+      })
+    : Map({ text: '*' })
+
+  if (query.sorts) {
+    const sorts = query.sorts.map(sort => {
+      const splitIndex = sort.lastIndexOf(',')
+      return {
+        propertyName: sort.substring(0, splitIndex),
+        sortOrder: sort.substring(splitIndex + 1, sort.length),
+      }
     })
+    filterMap = set(filterMap, 'sortOrder', sorts)
   }
 
-  return Map({ text: '*' })
+  filterMap = set(filterMap, 'sources', query.sources)
+  return filterMap
 }
 
 const createQuery = filterMap => {
   let sorts = get(filterMap, 'sortOrder')
-  if (sorts) {
-    sorts = sorts.map(
-      ({ propertyName, sortOrder }) => `${propertyName},${sortOrder}`
-    )
-  }
+  sorts = sorts
+    ? sorts.map(({ propertyName, sortOrder }) => `${propertyName},${sortOrder}`)
+    : null
+  let sources = get(filterMap, 'sources')
+
   return {
     filterTree: toFilterTree(filterMap),
-    sources: get(filterMap, 'sources'),
+    sources: sources ? sources : null,
```

Comment:
:pencil2: could also be written 
```suggestion
    sources: sources || null,
```